addon/components/array-container.js

```diff
@@ -134,6 +206,55 @@ export default Component.extend(PropTypeMixin, {
       this.notifyParentOfNewItem(newItem, index)
     },
 
+    onChange (bunsenId, value) {
+      const autoAdd = this.get('cellConfig.item.autoAdd')
+      const clearingValue = [undefined, null, ''].indexOf(value) !== -1
```

Comment:
Can we use _.contains() for cases like this? (e.g. `_.contains([undefined, null, ''], value)`)